crates/primitives/src/constants/mod.rs

```diff
@@ -61,6 +61,9 @@ pub const EIP1559_DEFAULT_BASE_FEE_MAX_CHANGE_DENOMINATOR: u64 = 8;
 /// Elasticity multiplier as defined in [EIP-1559](https://eips.ethereum.org/EIPS/eip-1559)
 pub const EIP1559_DEFAULT_ELASTICITY_MULTIPLIER: u64 = 2;
 
+/// Size of blobs in bytes as defined in [EIP-4844](https://eips.ethereum.org/EIPS/eip-4844)
+pub const EIP4844_GAS_PER_BLOB: u64 = 2u64.pow(17);
```

Comment:
ah we have this already:

https://github.com/paradigmxyz/reth/blob/838e6179ffc1acf075ab761e6736adcabaad0708/crates/primitives/src/constants/eip4844.rs#L12-L12

Reply:
Constant updated.